tests/python/unittest/test_ndarray.py

```diff
@@ -736,6 +736,14 @@ def test_output():
     assert_almost_equal(out.asnumpy(), ones.asnumpy() * 2)
     arange_out = mx.nd.arange(0, 20, dtype='int64')
     assert_almost_equal(arange_out.asnumpy(), np.arange(0, 20))
+    N_array = np.random.randint(1, high=3, size=3)
+    M_array = np.random.randint(1, high=3, size=3)
+    k_array = np.random.randint(-5, high=5, size=3)
```

Comment:
Could you run some more trials?
At least locally & with larger ranges. Just to verify if there are edge cases

Reply:
Also I think there is the case where M is 0